Formula/c/cortexso.rb

```diff
@@ -0,0 +1,38 @@
+require "language/node"
+
+class Cortexso < Formula
+  desc "Drop-in, local AI alternative to the OpenAI stack"
+  homepage "https://jan.ai/cortex"
+  url "https://github.com/janhq/cortex.git",
+      tag:      "v0.4.16",
+      revision: "0ec7db4dc0d6724dae76787f0f2a3e56a0ccdd7d"
```

Comment:
Do we need a Git checkout here? Can we use a tarball?

Reply:
This version does not work yet, but we will update it in a future release.